node/Cargo.toml

```diff
@@ -12,15 +12,15 @@ edition = "2021"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [build-dependencies]
-substrate-build-script-utils = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.30" }
+substrate-build-script-utils = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.32" }
 
 [[bin]]
 name = "trappist-collator"
 path = "src/main.rs"
 
 [dependencies]
 async-trait = "0.1.57"
-clap = { version = "3.2.17", features = ["derive"] }
+clap = { version = "4.0.9", features = ["derive"] }
 derive_more = "0.99.2"
```

Comment:
Looks like this can be removed, see [this](https://github.com/substrate-developer-hub/substrate-parachain-template/pull/138/files#diff-5dc0537169cb810865347de9185dae9ac7d231b7b5eb18eae2086c578dda133eL14).

Reply:
done